code/game/world.dm

```diff
@@ -313,9 +313,9 @@ var/world_topic_spam_protect_time = world.timeofday
 
 	s += "<b>[station_name()]</b>";
 	s += " ("
-	s += "<a href=\"http://\">" //Change this to wherever you want the hub to link to.
+	s += "<a href=\"https://discord.com/invite/aXrM3SDyEw">" //Change this to wherever you want the hub to link to.
```

Comment:
...yeah, you're missing a backslash here to escape the quotation mark: 
`"<a href=\"https://discord.com/invite/aXrM3SDyEw\">"`

Reply:
Oh! I was playing around with it on my own code and couldn't quite figure out what I did wrong. 

Thank you!! 💜